Backend/purrfect-match-backend/src/main/java/com/restservice/MailController/MailRegistrationController.java

```diff
@@ -0,0 +1,33 @@
+package com.restservice.MailController;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.mail.MailException;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+import com.restservice.ContactPageModel.ContactPage;
+import com.restservice.MailService.MailService;
+
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+@RestController
+public class MailRegistrationController {
+
+	@Autowired
+	private MailService notificationService;
+
+	@CrossOrigin(origins = "http://localhost:3000")
+	@PostMapping("/contact")
+	public String send(@RequestBody @Valid ContactPage contact) {
```

Comment:
The @Valid annotation does not perform validation itself, instead it marks that validation checks may need to be recursively performed. Since ContactPage is a flat structure this does nothing.